src/fear_and_greed/cnn.py

```diff
@@ -1,19 +1,10 @@
 #! /usr/bin/env python3
 
 import datetime
-import os.path
-import tempfile
 import typing
-
 import requests
-import requests_cache
-
 from random import choice
 
-requests_cache.install_cache(
```

Comment:
Please don't just drop this — it's here for a reason. At the very least, we should make it configurable whether these responses are cached. 

But why are you removing it? At first glance it doesn't seem to be incompatible with the new functionality you're adding.

Reply:
This was a mistake, I have rolled back the deletions